packages/vite-plugin-cloudflare/src/plugin-config.ts

```diff
@@ -11,26 +11,40 @@ import type {
 
 export type PersistState = boolean | { path: string };
 
-interface PluginWorkerConfig {
-	configPath: string;
+interface BaseWorkerConfig {
 	viteEnvironment?: { name?: string };
 }
 
-export interface PluginConfig extends Partial<PluginWorkerConfig> {
-	auxiliaryWorkers?: PluginWorkerConfig[];
+interface EntryWorkerConfig extends BaseWorkerConfig {
+	configPath?: string;
+}
+
+interface AuxiliaryWorkerConfig extends BaseWorkerConfig {
+	configPath: string;
+}
```

Comment:
There are a lot of Config types!

Given that we have WorkerConfig stuff below, I think it would be cleaner to call these something else.

E.g.

```suggestion
interface EntryWorkerPluginConfig extends BaseWorkerPluginConfig {
	configPath?: string;
}

interface AuxiliaryWorkerPluginConfig extends BaseWorkerPluginConfig {
	configPath: string;
}
```

Reply:
I agree that the types and there names need a cleanup but I think this needs to happen across both the `plugin-config` and `workers-configs` files. Is it OK if we don't do this in this PR?

I would rather keep the names of the types at the top of this file (`PluginConfig` and above) the same as they are the only public facing types and are referenced in the docs.